### DISCUSSION TITLE
Vercel has missing module? deploy failing

### DISCUSSION THREAD

**Discussion by Marfuen** [vercel/community#1484] [Help] [answered]
### Summary

I keep getting this error from vercel on deploy

Cannot find module '@vercel/lambda-node-bridge'

Seems like this is something on their end? How can I resolve, I can't deploy anything to production.

**Comment by thomashwi** [comment]
I am getting theese same errors during deployment

**Reply by thomashwi** [reply]
Was probably a passing issue, deploys running ok again now 👍

**Reply by Marfuen** [reply]
@thomashwi oh nice

**Reply by SkyM** [reply]
I am also getting deployment failures with this error message.

**Comment by amyegan** [comment, marked as answer]
The team looked into the cause of this issue and it is resolved now. Please retry and let me know if you still have any issues with it 🙏
